src/App.css

```diff
@@ -121,3 +121,97 @@ a {
   text-decoration: underline;
   text-underline-position: under;
 }
+
+
+
+
+
+
+body {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  min-height: 568px;
+}
+
+.game {

```

Comment:
@splendid-matt-derry how do you feel like splitting this out into a game sass file apposed to in the main app.css